### ISSUE TITLE
Brainrot is an annoying asshole.

### ISSUE THREAD

**Issue by killer653** [HippieStationArchive/HippieStation13-1#104]
So, at approx 7:10 P.M, 30 minutes before this post, Brainrot gets loose. Ok, no problem. Contact only & easy cure... Or so we thought.

Chemistry begins working on alkyzine, but shit is going ape. I go into chemistry and take 50u of alkysine for myself and pop a few pills and inject some of it. Ok, I should be cured. Nope. Inject some more, nope.avi. Go to viro and create a anti-bodies-metabolism virus and distribute it, 7 minutes left until shuttle. NOPE.AVI. Still not cured, even though the virus is at 5/5 and FINALLY it makes antibodies. Meanwhile, brainrot completely destroyed the station and everything is going to hell. All because of one guy with a brainrot bottle.

TL;DR: Brainrot is uncurable unless you actually work at viro, even though the cure is alky but it dosn't work.

**Comment by killer653** [comment]
Alkysine still dosn't fix brainrot. This bug is confirmed. Can someone work on it?

**Comment by AliiTea** [comment]
You 100% suuuuuuuuuuuuuuure it doesn't fix it?

**Comment by Kokojos** [comment]
Confirmed.

**Comment by AliiTea** [comment]
Rippers

**Comment by lcass** [comment]
ill have a look if I have some time.
